### PR TITLE
Update part of website 

### PR DESCRIPTION
**Pull request by vtraag**
DO NOT MERGE YET.

I already started editing the website. I already wanted to share this with you, so that it is clear that work is already being done. 

I still have to include the the installation instructions for Windows. I will do that later (so the link to those specific instructions for Windows is not yet working). @ntamas, I believe you said I could simply use Markdown, but wouldn't it be easier if I would just add the `compilations-windows.html` as I've included it now?

By the way, the release source code now refers to nightly builds. There are two things related to this: (1) we should link specifically to the released code; and (2) will we indeed provide nightly builds? Regarding nightly builds, perhaps it is best to simply always provide a source package for the latest master, but also clearly indicate the version (instead of simply nightly). That way, any reported issue can always be traced back more easily to an exact version.

Regarding the version that is reported, `{% include igraph-cversion %}` is now used for that, but on the website it simply seems to be replaced by `master`, not by the actual version, `0.8.0-pre+9b27241` (or whatever version exactly). This should be corrected still

### CODE REVIEW

**Comment by ntamas** [comment]
> DO NOT MERGE YET.

You can use [draft pull requests](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to indicate this.

> I believe you said I could simply use Markdown, but wouldn't it be easier if I would just add the compilations-windows.html as I've included it now?

I believe you can add HTML files directly to a Jekyll site, but you need to remove the HTML frontmatter and backmatter and include the body only. See `c/index.html` as an example.

> By the way, the release source code now refers to nightly builds.

Yes, that's because I have replaced `CVERSION` and `PYVERSION` in `Makefile` to point to the master branch and not to a concrete tag. This is so I could see the most recent output when building the site. Before release, this has to be changed to `0.8.0`, the appropriate revisions have to be tagged in Github, and then the site has to be rebuilt. I'll take care of it when the release is due. (Note that you need to tag the release first in the main repos).

Right now I don't plan to set up nightly builds for the documentation; we can do that later.

**Comment by vtraag** [comment]
> You can use [draft pull requests](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to indicate this.

Great, didn't know about this!

> I believe you can add HTML files directly to a Jekyll site, but you need to remove the HTML frontmatter and backmatter and include the body only. See `c/index.html` as an example.

Yes, exactly, I will do that.

> 
> > By the way, the release source code now refers to nightly builds.
> 
> Yes, that's because I have replaced `CVERSION` and `PYVERSION` in `Makefile` to point to the master branch and not to a concrete tag. This is so I could see the most recent output when building the site. Before release, this has to be changed to `0.8.0`, the appropriate revisions have to be tagged in Github, and then the site has to be rebuilt. I'll take care of it when the release is due. (Note that you need to tag the release first in the main repos).

OK, great!

> 
> Right now I don't plan to set up nightly builds for the documentation; we can do that later.

Ah, no, I don't mean nightly build of the documentation. I mean nightly builds of the source code of [`igraph`](https://github.com/igraph/igraph) itself. The [`nightly`](https://github.com/igraph/igraph/blob/master/tools/nightly) build script is probably outdated, but it might be something that might make it easier for people to start working with the latest development version. That would also allow us to get faster feedback and bugreports in, also for beta versions for example. Anyway, this is not urgent; we can do that at some later time.

**Comment by szhorvat** [comment]
"Discourse discussiong forum" has a typo.

Instead of "Discourse discussion forum" I would just say "igraph support forum". Discourse is just the technology we happen to use, but it's not the point.

Could you please add a link to the public Gitter chatroom? https://gitter.im/igraph/igraph

**Comment by ntamas** [comment]
Aren't we closing down the Gitter chatroom in favour of the Discourse forum?

**Comment by szhorvat** [comment]
I thought that referred to the private chatroom.

I think that forums and chatrooms are different, so there is value in having both. But I do not have a strong opinion on keeping/closing it.

**Comment by vtraag** [comment]
I would prefer only keeping Discourse. Otherwise we run the risk of having information scattered across multiple sources. Feel free to push to this PR btw.

**Comment by szhorvat** [comment]
OK, it's decided then. No public Gitter room for now.

File: REQUIREMENTS.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ The igraph homepage is built using Jekyll. Install
 [Jekyll](https://jekyllrb.com) if you don't have it yet. Further requirements
 are listed below.
 
+You need so install the following RubyGems (e.g. using `gem install`):
```
Comment (ntamas) [inline REQUIREMENTS.md]: I wouldn't list all the dependencies; `gem install` takes care of installing all the required dependencies anyway, wouldn't it? I would simply say that the user needs to run `gem install` to install everything.

File: c/index.html [hunk 2]
```diff
@@ -14,12 +14,8 @@
             <a class="nav-link" href="#startc">Get started</a>
             <ul class="nav">
               <li class="nav-item"><a class="nav-link" href="#creq">Requirements</a></li>
-              <li class="nav-item"><a class="nav-link" href="#cinstall">Installation</a></li>
-              <li class="nav-item"><a class="nav-link" href="#cinstallosx">Installation on Mac OS X</a></li>
```
Comment (ntamas) [inline c/index.html]: The "Installation on Mac OS X" heading was removed - is that intentional? I'd keep it and simply rename "Mac OS X" to "macOS"

File: REQUIREMENTS.md (hunk 1 above)
Comment (vtraag) [reply REQUIREMENTS.md]: Ah, good to know! I went through the whole list because `make` complained each time :)

**Comment by ntamas** [comment]
Let me know when it's ready to merge.

**Comment by vtraag** [comment]
I just re-read it and did some minor editing. I also included the new `html` file in the `Makefile`, which I previously forgot apparently. As far as I'm concerned you can go ahead and merge it.

**Pull request merged**